zoomable/src/main/java/net/engawapg/lib/zoomable/ZoomState.kt

```diff
@@ -233,6 +240,99 @@ class ZoomState(
             }
         }
     }
+
+    /**
+     * Zoom to the position based on content coordinate.
+     * @param zoomTo zoom target position
+     */
+    suspend fun zoomToOnContentCoordinate(
+        zoomTo: Offset,
+        scale: Float = 3f,
+        animationSpec: AnimationSpec<Float> = tween(700),
+    ) = coroutineScope {
+        val fitContentSizeFactor = fitContentSize.width / contentSize.height
```

Comment:
@masato1230 
fitContentSizeFactor is calculated in width / height, but not width / width?

```suggestion
        val fitContentSizeFactor = fitContentSize.width / contentSize.width
```